### ISSUE TITLE
Lots of RESCOPED activity

### ISSUE THREAD

**Issue by jchannon** [pragbits/stash2slack#1]
Hi,

We tend to have quite a few PRs open in Stash and so when we merge a PR it then sends in data saying all the others have been RESCOPED.

Is there a way to turn the Rescoping notifications off as it spams our Slack channel? :smile: 

Thanks

**Comment by jbparker** [comment]
We are running into this as well.  Would love to know how to disable these events.

**Comment by pragbits** [comment]
Sure, we will add a configurable event selection list to the next version.

**Comment by jchannon** [comment]
Cool! Thanks

**Comment by jbparker** [comment]
Thanks @pragbits - I'm not sure of these events being anything but internal, created PR to skip these so feel free to ignore PR I just created if you know of them not being internal.

**Comment by pragbits** [comment]
Well, I've tried to take a look, but I could not find any stable way to ignore the rescoped events from the other pull requests and let the one pass through in question. I posted a question to the atlassian developers community (https://answers.atlassian.com/questions/15290871/rescopedetails-added-and-removed-always-0), but I did not get any replies so far. Currently, I leave the rescoped events disabled.
